setup.py

```diff
@@ -60,15 +60,13 @@ def is_requirement(line):
 setup(
     name='MongoDBProxy',
```

Comment:
In order to avoid conflicting with the [upstream MongoDBPRoxy](https://pypi.org/project/MongoDBProxy/) project on PyPI, you might need to change this to `openedx-mongodbproxy`